ci/cudf_pandas_scripts/pandas-tests/diff.sh

```diff
@@ -10,12 +10,13 @@
 GH_JOB_NAME="pandas-tests-diff / build"
 rapids-logger "Github job name: ${GH_JOB_NAME}"
 
-MAIN_ARTIFACT=$(rapids-s3-path)cuda12_$(arch)_py310.main-results.json
-PR_ARTIFACT=$(rapids-s3-path)cuda12_$(arch)_py39.pr-results.json
+PY_VER="39"
```

Comment:
same reason as here: https://github.com/rapidsai/cudf/pull/15485/files#r1556345903

Reply:
Can I open a follow-up for this one? I will need to do some testing here